easybuild/framework/easyconfig/tools.py

```diff
@@ -538,6 +539,28 @@ def review_pr(paths=None, pr=None, colored=True, branch='develop'):
         else:
             lines.extend(['', "(no related easyconfigs found for %s)\n" % os.path.basename(ec['spec'])])
 
+    labels = []
+    file_info = det_file_info(pr_files, download_repo_path)
+    if any(file_info['new_folder']):
+        labels.append('new')
+    if any(file_info['new_file_in_existing_folder']):
+        labels.append('update')
```

Comment:
@migueldiascosta This is copy-pasted from `github.py`, can we flesh this out into a small separate function that takes `file_info` as input?

Perhaps we later enhance the logic here somehow to add labels more intelligently (or use other labels).

Reply:
done